File: test/resources/HelloProphecy/prophecy_scala/pipelines/silver_customers_orders/code/src/main/scala/io/prophecy/pipelines/silver_customers_orders/graph/bronze_customers.scala
package io.prophecy.pipelines.silver_customers_orders.graph

import io.prophecy.libs._
import io.prophecy.pipelines.silver_customers_orders.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object bronze_customers {

  def apply(context: Context): DataFrame =
    context.spark.read.table("`scottdemoscala`.`bronze_customers`")

}

File: test/resources/HelloProphecy/prophecy/pipelines/silver_customers_orders/code/silver_customers_orders/graph/bronze_customers.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from silver_customers_orders.config.ConfigStore import *
from silver_customers_orders.udfs.UDFs import *

def bronze_customers(spark: SparkSession) -> DataFrame:
    return spark.read.table("`scottdemo`.`bronze_customers`")
